prime-router/build.gradle.kts

```diff
@@ -772,6 +770,7 @@ tasks.named<nu.studer.gradle.jooq.JooqGenerate>("generateJooq") {
 tasks.register("compile") {
     group = rootProject.description ?: ""
     description = "Compile the code"
+    dependsOn("generateVersionObject")
```

Comment:
I think resolves folks issue with the `Version` generated object not being present. If needs be, we add a further depedency for `compileKotlin` but I don't think that is required

Reply:
@JFisk42 @mkalish et al 

running the gradle task before cleanslate on the impacted commits does not resolve the issue

Reply:
K I need to push it in one more spot.  However, cleanslate shouldn't impact your ability to run master

Reply:
works now!